### DISCUSSION TITLE
during the first installation phase of the application, I get this error "Cannot be run from folder that contains non-ASCII characters in path", can you help me do what I need to do?

### DISCUSSION THREAD

**Discussion by Serkankarakus-blip** [winder/Universal-G-Code-Sender#1570] [Help]
during the first installation phase of the application, I get this error "Cannot be run from folder that contains non-ASCII characters in path", can you help me do what I need to do?
![Ekran görüntüsü 2021-03-25 053855](https://user-images.githubusercontent.com/69793995/112410575-72db8a80-8d2c-11eb-8399-e023a9b8b58f.png)

**Comment by breiler** [comment]
What is the complete directory path of your installation of UGS (ie: C:\Users\MyUser\Downloads\ugs).

**Reply by Serkankarakus-blip** [reply]
C:/ users /KARAKUŞ/ Dowloads/ugs

**Comment by huseyinctlbs** [comment]
> during the first installation phase of the application, I get this error "Cannot be run from folder that contains non-ASCII characters in path", can you help me do what I need to do?
> ![Ekran görüntüsü 2021-03-25 053855](https://user-images.githubusercontent.com/69793995/112410575-72db8a80-8d2c-11eb-8399-e023a9b8b58f.png)

Ş harfi yüzünden sıkıntı oluyor hocam. Dosyaları c: kök dizine taşıyarak dene.

**Reply by Serkankarakus-blip** [reply]
Teşekkür ederim düzeldi  hocam.

**Comment by nikman68** [comment]
same problem
C:\Users\ΝΙΚΟΣ\OneDrive\Desktop\ugs\ugsplatform-win\bin

**Reply by breiler** [reply]
Σ is a non ascii character, move to another path.

**Comment by nikman68** [comment]
Thanks a lot. Solved.
